src/animation/types.ts

```diff
@@ -1,23 +1,27 @@
 export type Primitive = number | string;
 
+interface BaseDriverOptions {
+  from?: Primitive;
```

Comment:
I just noticed, we only support `numer` type for `from`. Can you update this please ?